test/common/config/filesystem_subscription_impl_test.cc

```diff
@@ -27,6 +28,14 @@ TEST_F(FilesystemSubscriptionImplTest, InitialFile) {
   verifyStats(1, 1, 0, 0, 7148434200721666028);
 }
 
+// Validate that if we fail to set a watch, we get a sensible warning.
+TEST_F(FilesystemSubscriptionImplTest, BadWatch) {
+  updateFile("", true);
+  EXPECT_THROW_WITH_MESSAGE(
+      FilesystemEdsSubscriptionImpl(dispatcher_, "@/dev/null", stats_), EnvoyException,
```

Comment:
On macos:

test/common/config/filesystem_subscription_impl_test.cc:36: Failure
Expected equality of these values:
  "unable to add filesystem watch for file @/dev/null: No such file or directory"
  std::string(e.what())
    Which is: "invalid watch path @/dev/null"